Sources/SwiftRouting/Router/Router.swift

```diff
@@ -35,6 +35,10 @@ public final class Router: BaseRouter, @unchecked Sendable {
   public var isPresented: Bool {
     type.isPresented
   }
+  /// current number of routes
+  public var routeCount: Int {
+    path.count + (root != nil ? 1 : 0)
```

Comment:
we should always add 1,
`Root` can be nil because we can initialize a Router with a View directly